packages/terra-spacer/src/Spacer.scss

```diff
@@ -0,0 +1,325 @@
+:local {
+  .margin-top-none {
+    margin-top: 0;
+  }
+
+  .margin-top-small-minus-2 {
+    margin-top: var(--terra-spacer-margin-top-small-minus-2, 0.214rem); // 3px
+  }
+
+  .margin-top-small-minus-1 {
+    margin-top: var(--terra-spacer-margin-top-small-minus-1, 0.357rem); // 5px
```

Comment:
Update `0.357rem` to `0.3571428571rem`. The extra decimal values help ensure the rem computes to a whole value, 5px, instead of a sub-pixel value.

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/b2ec65a5ad9f9f5b81c9394939eb477f56b21774#diff-bbd694561d6e058afb7ec7501feecb1aR1